reference/orders.v3.yml

```diff
@@ -2253,9 +2253,6 @@ definitions:
         description: Reason for the refund.
         minLength: 0
         maxLength: 1000
-      item_id:
```

Comment:
is this a desire to change and require engineering work rather than a simple doc update ? 

I did a quick check and it seems like we do currently send `item_id` in the request for order level refund.

Reply:
We send it but it is only used here from https://github.com/bigcommerce/bigcommerce/blob/08d84bb67d250c7624502ed76d8e0bafb1a5cacf/app/Order/DomainModel/Refund/ItemsRefunder.php#L90

It's pulling the `order_id` from the url. 

We should probably update the linked code to use the order id 
Below is an example, the refund succeeds, the `item_id` does not exist 
<img width="1319" alt="Screen Shot 2021-05-07 at 11 14 06 am" src="https://user-images.githubusercontent.com/2238749/117384016-5b93dd80-af25-11eb-96ce-c401eb67eae5.png">

Reply:
@WillemHoman thanks for the explanation and raising a bug ticket for this. 

My guess is we had the field in the past for "consistency" reasons. If there's actually no need for that field at all as we have the `order_id` from url anyway, we should definitely explore whether we can simply remove it. 

All I tried to understand is whether this change can be merged and published now, or it needs to wait until some engineering work is done.

Reply:
I think it can be merged as-is, but we should address the `item_id` in the response payload as described above to make it the `order_id` regardless of whether `item_id` is passed in as part of the request payload. @bc-rogq ?